src/renderer/component/video/internal/player.jsx

```diff
@@ -20,6 +20,13 @@ class VideoPlayer extends React.PureComponent {
     this.togglePlayListener = this.togglePlay.bind(this);
   }
 
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.videoPause) {
+      this.refs.media.children[0].pause();
+      this.props.setVideoPause(false);
```

Comment:
is this (`this.props.setVideoPause(false);`) necessary? can we trigger this, instead, when they push "play" in app?